ruby/test/vaas_test.rb

```diff
@@ -172,6 +170,24 @@ def create(token = nil, timeout = nil)
           vaas.close
         end
       end
+      
+      specify 'upload_failed' do
+        skip("certificate verify failed (self-signed certificate) for vaas staging")
+        vaas, token = create
+        message = {"url" => "https://upload.staging.vaas.gdatasecurity.de/upload", "upload_token" => "invalid_token"}
+        Async do
+          random_text = (0...8).map { (65 + rand(26)).chr }.join
+          File.open("test.txt", "w") { |f| f.write(random_text) }
+
+          vaas.connect(token)
+          assert_raises VAAS::VaasUploadError do
+            vaas.upload(message, "./test.txt").wait
+          end
+        ensure
+          vaas.close
+        end
+      end
```

Comment:
We removed the old upload failed test as it didn't have much use. The upload endpoint used here doesn't exist anymore and will get nothing but 404 in any case. That's also why it gets a self-signed cert. I think we can remove this entirely.

Reply:
I see. I didn't get the test running so I decided to ignore it, as I didn't want to touch any of the logic of any of the SDKs. I'll remove the test.